monitor/dbt_project/macros/get_new_alerts_where_clause.sql

```diff
@@ -10,7 +10,7 @@
     {%- set row_count = elementary.get_row_count(alerts_model_relation) %}
```

Comment:
I think instead of checking the row count we can check if last_alert_time_with_backfill is not none (we need to check what the macro result_value returns if it gets empty results). This will save a redundant query